Library/Homebrew/sandbox.rb

```diff
@@ -127,7 +127,7 @@ def exec(*args)
         end
       end
 
-      if @failed && ARGV.verbose?
```

Comment:
@MikeMcQuaid  `Sandbox` runs in a separate thread, replacing this with `Homebrew.args.verbose?` does not work as `CLI:Parser` is not instantiated yet - so global args dependant on environment vars will not be set.

Reply:
We could fix this by directly checking if `HOMEBREW_VERBOSE` is set or decoupling global vars from `CLI::Parser` instantiation.

Reply:
> * Directly checking if `HOMEBREW_VERBOSE` is set instead of `Homebrew.args.verbose?`

This seems reasonable for a one-time thing 👍